README.rst

```diff
@@ -1,76 +1,51 @@
-
-2. Use node v12.x.
-
-   The micro-frontend build scripts support node 12.  Using other major versions of node *may* work, but is unsupported.  For convenience, this repository includes an .nvmrc file to help in setting the correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
-
-3. Install npm dependencies:
-
-  ``cd frontend-template-application && npm install``
-
-4. Update the application port to use for local development:
-
-   Default port is 8080. If this does not work for you, update the line `PORT=8080` to your port in all .env.* files
+Development
+------------
 
-5. Start the dev server:
+Start the development server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In this project, install requirements and start the development server by running:
 
-  ``npm start``
+.. code-block::
 
-The dev server is running at `http://localhost:8080 <http://localhost:8080>`_ or whatever port you setup.
+   cd frontend-app-skills
+   npm install
+   npm start # The server will run on port 8080
 
-Making Your New Project's README File
--------------------------------------
+Once the dev server is up visit, http://localhost:8080
 
-Move the file ``README-template-frontend-app.rst`` to your project's ``README.rst`` file. Please fill out all
-the sections - this helps out all developers understand your MFE, how to install it, and how to use it.
+Environment Variables
+---------------------------------
 
-Project Structure
------------------
+In order to run the Skills Builder, you'll need to add Algolia keys to the ``.env.development`` file. If you have access to the `edx-internal repo`_, you can copy the ones for the Skills MFE's stage configuration. Alternatively, reach out to **@openedx/2u-aperture**.
 
-The source for this project is organized into nested submodules according to the ADR `Feature-based Application Organization <https://github.com/openedx/frontend-template-application/blob/master/docs/decisions/0002-feature-based-application-organization.rst>`_.
+.. _edx-internal repo: https://github.com/edx/edx-internal
 
-Build Process Notes
+Development Roadmap
 -------------------
 
-**Production Build**
-
-The production build is created with ``npm run build``.
+The following is a list of current short-term development targets, in (rough) descending order of priority:
+  
+* `edX Skills Builder - Fast Follows`_
 
-Internationalization
---------------------
+* `edX Skills Builder - MVP Tech Debt`_
 
-Please see `edx/frontend-platform's i18n module <https://edx.github.io/frontend-platform/module-Internationalization.html>`_ for documentation on internationalization.  The documentation explains how to use it, and the `How To <https://github.com/openedx/frontend-i18n/blob/master/docs/how_tos/i18n.rst>`_ has more detail.
+.. _edX Skills Builder - Fast Follows: https://2u-internal.atlassian.net/browse/APER-2299
+.. _edX Skills Builder - MVP Tech Debt: https://2u-internal.atlassian.net/browse/APER-2387
```

Comment:
I'm not sure we should have this in the README. It is likely to be good information for only a short amount of time. This could be stale quickly as things move and change around this project. I'd omit if possible.

Reply:
@MaxFrank13 let's just make a single wiki page for "development roadmap" and link to it here.

Reply:
Here's a stub: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3764944925/Skills+MFE+Development+Roadmap

Reply:
Okay I will move all of this to the Roadmap doc and link that instead 👍